src/aks-safeguards/azext_aks_safeguards/_help.py

```diff
@@ -0,0 +1,11 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps  # pylint: disable=unused-import
```

Comment:
do we need to manually write help msg for aaz module?

Reply:
No idea - probably? My guess is that this is a stub generated by aaz